helm/charts/hpe-csi-driver/README.md

```diff
@@ -38,6 +38,7 @@ The following table lists the configurable parameters of the chart and their def
 | disableNodeConfiguration  | Disables node conformance and configuration.`*`                        | false            |
 | disableNodeGetVolumeStats | Disable NodeGetVolumeStats call to CSI driver.                         | false            |
 | disableNodeMonitor        | Disables the Node Monitor that manages stale storage resources.        | false            |
+| disableHostDeletion       | Disables host deletion during volume unpublish                         | false            |
```

Comment:
Can you change this to `Disables host deletion by the CSP when no volumes are associated with the host.`? "volume unpublish" is an implementation detail users won't care about. Please change throughout the PR.

Reply:
Done